src/pages/WorldsDashboard/WorldsDashboard.tsx

```diff
@@ -26,6 +31,33 @@ export const WorldsDashboard: React.FC = () => {
     history,
   ]);
 
+  const user = useUser();
+
+  const worlds = useOwnWorlds(user.userId);
+
+  const hasWorlds = !!worlds.length;
+
+  const renderWelcomePage = useMemo(
+    () => (
+      <div className="WorldsDashboard__messages-container">
+        <AdminShowcaseTitle>Let’s create a world</AdminShowcaseTitle>
+        <AdminShowcaseSubTitle>It’s fast and easy</AdminShowcaseSubTitle>
+      </div>
+    ),
+    []
+  );
+
+  const renderWorldsList = useMemo(
```

Comment:
render (verb) worlds list- method, usually ending with `()` when used

rendered (adjective) worlds list - object, named and with specifiers,, not followed by `()` in `{hasWorlds ? renderWorldsList : renderWelcomePage}`